content/en/docs/reference/glossary/ambient.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Ambient
+test: n/a
+---
+
+Ambient refers to a {{< gloss >}}data plane mode{{< /gloss >}} consisting of a per-node per-namespace component.
+Ambient mode is an alternative to a {{< gloss >}}sidecar{{< /gloss >}} based deployment.
```

Comment:
Glossary entries don't support second level glossary entries, so they need to use links instead.

```suggestion
Ambient refers to a [data plane mode](/docs/reference/glossary/#data-plane-mode) consisting of a per-node per-namespace component.
Ambient mode is an alternative to a [sidecar](/docs/reference/glossary/#sidecar) based deployment.
```
(/docs/reference/glossary/#service)